Add `International Convert Cup 2024` wiki article (#12761)
ppy/osu-wiki: wiki/Tournaments/IntlConvertCup/2024/en.md

| 31 | Convert noob | 29.50 |
| 32 | Fate | 29.75 |
| 33 | Enterprise | 31.25 |
| 34 | For fun gamers | 31.38 |
| 35 | 3 dif digits | 33.63 |
| 36 | looking for a team name | 35.00 |
| N/A | abubeshki | *DNP* |
| N/A | Albacete x ICC | *DNP* |

## Ruleset

### General rules

1. Match lobbies across the tournament will adhere to the following room settings:
   - Team Mode: `Head-to-head`
   - Win Condition: [`ScoreV2`](/wiki/Gameplay/Score#scorev2)
2. The mappools for each round will be announced by the tournament management in advance before the actual matches take place.
3. Match schedules will be predetermined by the tournament management. If there are any teams who are unable to attend the current schedule for any reason, all other affected teams may apply and settle for a reschedule at the `#reschedule` channel in the tournament's Discord server.
4. A referee will create a multiplayer room 10 minutes in advance and will start to send out invites.
5. If a team does not show up or is unable to field in a sizeable 2v2 roster within **10 minutes** of the start time, their opponent gets to win by default.
6. If no staff or referee is available, the match will be postponed.

> 7. **NoFail will be enforced in all beatmaps.** This is to ensure that the points are to be awarded more fairly towards teams who perform better in general during the course of the beatmap regardless of their remaining health at the end.

Reviewer: since these are spaced elsewhere in this file

```suggestion
7. **No Fail will be enforced in all beatmaps.** This is to ensure that the points are to be awarded more fairly towards teams who perform better in general during the course of the beatmap regardless of their remaining health at the end.
```

8. If a player disconnects, it will be treated as if they had failed the beatmap.
   - A match can be rematched for disconnects that occur within a few seconds after the beatmap has been started by the referee.
9. Lag is not a valid reason to nullify a beatmap.
10. If any problems during the match occur, the tournament management will make a decision based on the referee's report.
11. It is expected that all players be polite and respectful to each other. Penalties will be given upon violation.
    - If a player is found to be engaging in an act that is deemed to be distasteful or provocative, the corresponding player or their team may be disqualified right away from the tournament and/or blacklisted from future iterations of the tournament by the tournament management.
    - Usage of any tools or programs that are against the [osu! community rules](/wiki/Rules#community-rules) is strictly prohibited and will be straight up reported to the osu! team at will.

### Tournament registration

1. In order to participate, players are required to form a team and register into the tournament as a team through [the tournament's website](https://wybin.xyz/tournaments/icc2024).

>    - Each team may only enlist one player ranked #1 - #50 and one player ranked #50 - #100 according to [osupepe.com's convert ranking](osupepe.com) at the time of registration in their roster.

Reviewer: ```suggestion
   - Each team may only enlist one player ranked #1 – #50 and one player ranked #50 – #100 according to [osupepe.com's convert ranking](osupepe.com) at the time of registration in their roster.
```

   - The winner of the `!roll` gets to determine who gets the first pick and the second ban.
   - The loser of the `!roll` gets the opposite by default.
   - This rule does not apply in the Qualifier lobbies.
2. Each team has to **ban one beatmap** from the corresponding mappool. These beatmaps will not be allowed to be picked by any team during the entire match. 
   - Barring the tiebreaker, there are no restrictions as to which maps may and may not be banned in a match.
   - Banning does not apply in the Qualifier lobbies.
3. Each team will be given a chance to pick **one warm-up beatmap** to be played in the lobby.
   - Playing a warm-up beatmap is not mandatory, and teams may elect to skip their warm-up pick should they wish to.
4. Teams are expected to exercise common sense in pick time windows.
   - Teams are given 2 minutes to pick a beatmap and 2 minutes to press the Ready button on their client.
   - If a team is unable to come up with a pick within their time window, the pick will be chosen by random by the referee using the `!roll X` command.
5. Not counting the Qualifiers, each stage of the tournament will feature a **Mixed Mod pool** that goes as follows: 

>    - The Mixed Mod pool will be played with FreeMod activated.

Reviewer: ```suggestion
   - The Mixed Mod pool will be played with Free Mod activated.
```
